src/internal.h

```diff
@@ -29,5 +29,9 @@ char *internalRemoteUri(bool, const char *, unsigned short, const char *, const
 const char *internalHostname(void);
 int internalHostnameIs(const char *);
 
+/// whether the given request URL path points to a cache manager (not
+/// necessarily running on this Squid instance)
+bool ForSomeCacheManager(const SBuf &);
```

Comment:
This might be better placed in the class CacheManager API.  I do not insist on this.

Reply:
Yeah, it is difficult to say for sure. I think this is the best location _for now_ because the function is used, in part, by the code that does not need to know any of the cache manager APIs. This is also where many other similar functions are already located. This function _implementation_ also benefits from being close to its sister functions -- it is easier to keep that code in sync that way.

We can and perhaps should refactor this code to allow individual test functions to be separated into their own modules, but it is best done in a dedicated PR, affecting all of these functions. I would not be surprised if proper refactoring will completely remove this pair of files, but all that is out of scope here.